systemtest/gencorpora/catbulk.go

```diff
@@ -113,18 +171,31 @@ func handleReq(writer io.Writer) http.HandlerFunc {
 
 			mockResp := esutil.BulkIndexerResponse{}
 			scanner := bufio.NewScanner(reader)
+			var stat docsStat
 			for count := 0; scanner.Scan(); count++ {
-				fmt.Fprintln(writer, scanner.Text())
+				n, err := fmt.Fprintln(writer, scanner.Text())
+				stat.bytes += n
+
 				// create mock bulk response considering all request items
 				// exist with action_and_metadata followed by source line
 				if count%2 == 0 {
+					respStatus := http.StatusOK
+					if err != nil {
+						respStatus = http.StatusInternalServerError
+						stat.count++
```

Comment:
Is it intentional that we are counting failed docs twice? (stat.count++ here and below)
If so, why?

Reply:
Apologies, I was in between commits and pushed a mess 😓 . I have fixed this now.